From: Irfan Mohammad <irfan@graphiant.com>
Date: Sun, 11 Dec 2022 01:22:48 +0000
Subject: sysrepo: log on connection destroy

Add a log to indicate when a connection is destroyed,
mirroring existing logs when these objects are created.

Remove unneccessary logs for session start or stop
There is really no added benefit of logging session start and session
stop.

Almost all locking is tracked with connection ID. sysrepo-monitoring can
be used to dump the sessions which hold any module ds locks.
---
 src/sysrepo.c | 7 ++-----
 1 file changed, 2 insertions(+), 5 deletions(-)

diff --git a/src/sysrepo.c b/src/sysrepo.c
index 7ff6e83..2c766cf 100644
--- a/src/sysrepo.c
+++ b/src/sysrepo.c
@@ -411,6 +411,8 @@ sr_disconnect(sr_conn_ctx_t *conn)
         return sr_api_ret(NULL, err_info);
     }
 
+    SR_LOG_INF("Connection %" PRIu32 " destroyed", conn->cid);
+
     /* free attributes */
     sr_conn_free(conn);
 
@@ -723,11 +725,6 @@ _sr_session_start(sr_conn_ctx_t *conn, const sr_datastore_t datastore, sr_sub_ev
         goto error;
     }
 
-    if (!event) {
-        SR_LOG_INF("Session %" PRIu32 " (user \"%s\", CID %" PRIu32 ") created.", (*session)->sid, (*session)->user,
-                conn->cid);
-    }
-
     return NULL;
 
 error:
